lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/domain/dva/request/DvaPayload.java

```diff
@@ -184,43 +157,38 @@ public void setIssueNumber(String issueNumber) {
         this.issueNumber = issueNumber;
     }
 
+    public String getIssuerId() {
+        return issuerId;
+    }
+
+    public void setIssuerId(String issuerId) {
+        this.issuerId = issuerId;
+    }
+
     @Override
     public String toString() {
-        return "DcsPayload{"
-                + "correlationId="
-                + correlationId
-                + ", requestId="
+        return "DvaPayload{"
+                + "requestId="
                 + requestId
                 + ", timestamp='"
                 + timestamp
                 + '\''
-                + ", licenceNumber='"
-                + licenceNumber
-                + '\''
-                + ", driverNumber='"
-                + driverNumber
+                + ", driverLicenceNumber='"
+                + driverLicenceNumber
                 + '\''
-                + ", postcode='"
+                + ", address='"
                 + postcode
                 + '\''
-                + ", clientId='"
-                + clientId
-                + '\''
                 + ", surname='"
```

Comment:
Should this not be family name

Reply:
local variable names that store the values are all the same as existing/DCS, the values then get mapped to the correct JSONProperties in the payload. To me this way is clearer/inline with what's already there but if it's more confusing then I can change the fieldNames to match the JSON field names

Reply:
Its not the naming thats the issue here its the inconsistency i.e
We have
surname=surname
givenNames=forenames

Should all be 
fieldname=fieldname

or 

propertyname=fieldname

Anything in between is confusing

Reply:
Ok fair enough, if you're happy though I'm gonna keep postcode being mapped to address to save confusion around street names etc. which are not included in this address field